guidelines/REUSE.md

```diff
@@ -92,14 +147,14 @@ _Assuming file https://my.company.com/definitions/Model.json_
       "format": "int64"
     },
     "tag": {
-      "description": "a complex, shared property.  Note the absolute reference",
+      "description": "A complex, shared property.  Note the absolute reference",
```

Comment:
I know this was in the original example as well, but the `description` is obviously ignored and does not override the value inside the reference (as previously mentioned in the doc). I get this is used here for documentation to the readers, but it may also send the wrong message. I'll merge as-is, but we may consider removing it after all.